content/acra/configuring-maintaining/general-configuration/acra-backup.md

```diff
@@ -108,6 +108,25 @@ Should be provided only with `--keystore_encryption_type=<kms_encrypted_master_k
 {{< /hint >}}
 
 
+### KMS
```

Comment:
The same here. Maybe it's some git collision.

Reply:
i don't how and why I copied it... but i removed